bool set_value
---
bool return_status